.github/workflows/scripts/scanner-update-nvd-legacy.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+"""
+Fetch NVD data from the JSON 1.1 feed archives, parse them into API 2.0
+schema and save into files.
+"""
+
+import gzip
+import json
+import logging
+import os
+from urllib import request
+from urllib import parse
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(levelname)s: %(name)s %(message)s")
+
+
+def convert_cve_feed_to_api(cve_feed):
+    """Convert NVDCVEFeedJSON10DefCVEItem to CVEAPIJSON20DefCVEItem."""
+
+    # Required fields.
+
+    cve_api = {"id": cve_feed['cve']['CVE_data_meta']['ID'],
+               "published": cve_feed['publishedDate'],
+               "lastModified": cve_feed['lastModifiedDate'],
```

Comment:
the time formats are not the same, so this may cause issues

Reply:
actually that may not matter based on how we are using this

Reply:
Yes! Thanks.

Reply:
Wait, can you give me an example? They seem to be  ISO 8601 dates.

Reply:
From https://services.nvd.nist.gov/rest/json/cves/2.0/?noRejected I see one example:

```
"published":"1988-10-01T04:00:00.000","lastModified":"2019-06-11T20:29:00.263"
```

A random one from 2024.json:

```
"publishedDate" : "2024-02-14T18:15Z",
"lastModifiedDate" : "2024-02-15T06:23Z"
```

Reply:
They are both ISO 8601 dates. They should be parsed the same, except the 2.0 shows milliseconds while 1.1 does not, but I don't think that impacts anything (the milliseconds resolution).